map_gui/src/render/agents.rs

```diff
@@ -219,22 +222,61 @@ impl UnzoomedAgents {
         }
     }
 
-    pub fn make_horiz_viz_panel(&self, ctx: &mut EventCtx) -> Widget {
+    /// The details are (car, bike, bus, pedestrian)
+    pub fn make_horiz_viz_panel(
+        &self,
+        ctx: &mut EventCtx,
+        details: (Widget, Widget, Widget, Widget),
+    ) -> Widget {
+        let (car_details, bike_details, bus_details, pedestrian_details) = details;
         Widget::custom_row(vec![
-            Toggle::colored_checkbox(ctx, "Car", self.car_color, self.cars).margin_right(24),
-            Toggle::colored_checkbox(ctx, "Bike", self.bike_color, self.bikes).margin_right(24),
-            Toggle::colored_checkbox(ctx, "Bus", self.bus_color, self.buses_and_trains)
-                .margin_right(24),
-            Toggle::colored_checkbox(ctx, "Walk", self.ped_color, self.peds).margin_right(8),
+            Widget::col(vec![
+                Toggle::colored_checkbox(ctx, "Car", self.car_color, self.cars),
+                car_details,
+            ])
+            .margin_right(24),
+            Widget::col(vec![
+                Toggle::colored_checkbox(ctx, "Bike", self.bike_color, self.bikes),
+                bike_details,
+            ])
+            .margin_right(24),
+            Widget::col(vec![
+                Toggle::colored_checkbox(ctx, "Bus", self.bus_color, self.buses_and_trains),
+                bus_details,
+            ])
+            .margin_right(24),
+            Widget::col(vec![
+                Toggle::colored_checkbox(ctx, "Walk", self.ped_color, self.peds),
+                pedestrian_details,
+            ])
+            .margin_right(8),
         ])
     }
 
-    pub fn make_vert_viz_panel(&self, ctx: &mut EventCtx) -> Widget {
+    /// The details are (car, bike, bus, pedestrian)
+    pub fn make_vert_viz_panel(
```

Comment:
Is that this screen? The width jitter is pretty rough:

https://user-images.githubusercontent.com/217057/111011600-cc1be500-834e-11eb-8a9e-77424ac96713.mov

I can take a look at that in my upcoming work if you'd like to hand it off.

Reply:
Yep, this is it. I don't have immediate ideas other than left-padding the numbers with spaces, assuming a max of 10k, and either using a fixed-width font or calculating+hardcoding the width for the widest numbers.